io.github.hakuneko.HakuNeko.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>io.github.hakuneko.HakuNeko</id>
+	<launchable>io.github.hakuneko.HakuNeko.desktop</launchable>>
```

Comment:
```suggestion
	<launchable type="desktop-id">io.github.hakuneko.HakuNeko.desktop</launchable>>
```

Reply:
I added the launchable type you suggested.